scripts/generateChangedIconsCommentMarkup.mjs

```diff
@@ -73,11 +73,26 @@ const changeFiles3pxStrokeImageTags = getImageTagsByFiles(
   () => `${BASE_URL}/stroke-width/3`,
 ).join('');
 
-const changeFilesLowDPIImageTags = getImageTagsByFiles(
+const changeFilesLowDPIImageTags18 = getImageTagsByFiles(
+  changedFiles,
+  () => `${BASE_URL}/dpi/18`,
```

Comment:
Is `16px` not a more conventional size to use than `18px`?
```suggestion
  () => `${BASE_URL}/dpi/16`,
```

Reply:
Oh you are right, I thought shadcn was 18x18.
<img width="330" alt="billede" src="https://github.com/user-attachments/assets/0934cbc0-1089-425e-922d-63388d1af80c" />